src/data/symbol/symbol.tsv

```diff
@@ -1957,85 +2056,85 @@ POS	CHAR	Reading (space separated)	description	additional description	category	m
 記号	𛄡	へんたいがな いぇ	変体仮名
 記号	𛄡	へんたいがな え	変体仮名
 記号	𛄢	へんたいがな う	汙の変体仮名
-記号	あ゙	あ だくてん	濁点付き仮名
-記号	ぁ゙	あ ぁ だくてん	濁点付き仮名
-記号	い゙	い だくてん	濁点付き仮名
-記号	ぃ゙	い ぃ だくてん	濁点付き仮名
-記号	ぅ゙	う ぅ だくてん	濁点付き仮名
-記号	え゙	え だくてん	濁点付き仮名
-記号	ぇ゙	え ぇ だくてん	濁点付き仮名
-記号	お゙	お だくてん	濁点付き仮名
-記号	ぉ゙	お ぉ だくてん	濁点付き仮名
-記号	っ゙	づ っ だくてん	濁点付き仮名
-記号	な゙	な だくてん	濁点付き仮名
-記号	に゙	に だくてん	濁点付き仮名
-記号	ぬ゙	ぬ だくてん	濁点付き仮名
-記号	ね゙	ね だくてん	濁点付き仮名
-記号	の゙	の だくてん	濁点付き仮名
-記号	ま゙	ま だくてん	濁点付き仮名
-記号	み゙	み だくてん	濁点付き仮名
-記号	む゙	む だくてん	濁点付き仮名
-記号	め゙	め だくてん	濁点付き仮名
-記号	も゙	も だくてん	濁点付き仮名
-記号	や゙	や だくてん	濁点付き仮名
-記号	ゃ゙	や ゃ だくてん	濁点付き仮名
-記号	ゆ゙	ゆ だくてん	濁点付き仮名
-記号	ゅ゙	ゆ ゅ だくてん	濁点付き仮名
-記号	よ゙	よ だくてん	濁点付き仮名
-記号	ょ゙	よ ょ だくてん	濁点付き仮名
-記号	ら゙	ら だくてん	濁点付き仮名
-記号	り゙	り だくてん	濁点付き仮名
-記号	る゙	る だくてん	濁点付き仮名
-記号	れ゙	れ だくてん	濁点付き仮名
-記号	ろ゙	ろ だくてん	濁点付き仮名
-記号	わ゙	わ だくてん	濁点付き仮名
-記号	ゐ゙	ゐ だくてん	濁点付き仮名
-記号	ゑ゙	ゑ だくてん	濁点付き仮名
-記号	を゙	を だくてん	濁点付き仮名
-記号	ん゙	ん だくてん	濁点付き仮名
-記号	ア゙	あ だくてん	濁点付き仮名
-記号	ァ゙	あ ぁ だくてん	濁点付き仮名
-記号	イ゙	い だくてん	濁点付き仮名
-記号	ィ゙	い ぃ だくてん	濁点付き仮名
-記号	ゥ゙	う ぅ だくてん	濁点付き仮名
-記号	エ゙	え だくてん	濁点付き仮名
-記号	ェ゙	え ぇ だくてん	濁点付き仮名
-記号	オ゙	お だくてん	濁点付き仮名
-記号	ォ゙	お ぉ だくてん	濁点付き仮名
-記号	ッ゙	づ っ だくてん	濁点付き仮名
-記号	ナ゙	な だくてん	濁点付き仮名
-記号	ニ゙	に だくてん	濁点付き仮名
-記号	ヌ゙	ぬ だくてん	濁点付き仮名
-記号	ネ゙	ね だくてん	濁点付き仮名
-記号	ノ゙	の だくてん	濁点付き仮名
-記号	マ゙	ま だくてん	濁点付き仮名
-記号	ミ゙	み だくてん	濁点付き仮名
-記号	ム゙	む だくてん	濁点付き仮名
-記号	メ゙	め だくてん	濁点付き仮名
-記号	モ゙	も だくてん	濁点付き仮名
-記号	ヤ゙	や だくてん	濁点付き仮名
-記号	ャ゙	や ゃ だくてん	濁点付き仮名
-記号	ユ゙	ゆ だくてん	濁点付き仮名
-記号	ュ゙	ゆ ゅ だくてん	濁点付き仮名
-記号	ヨ゙	よ だくてん	濁点付き仮名
-記号	ョ゙	よ ょ だくてん	濁点付き仮名
-記号	ラ゙	ら だくてん	濁点付き仮名
-記号	リ゙	り だくてん	濁点付き仮名
-記号	ル゙	る だくてん	濁点付き仮名
-記号	レ゙	れ だくてん	濁点付き仮名
-記号	ロ゙	ろ だくてん	濁点付き仮名
-記号	ヷ	わ だくてん	濁点付き仮名
-記号	ヸ	ゐ だくてん	濁点付き仮名
-記号	ヹ	ゑ だくてん	濁点付き仮名
-記号	ヺ	を だくてん	濁点付き仮名
-記号	ン゙	ん だくてん	濁点付き仮名
-記号	か゚	びだくおん か が	鼻濁音
-記号	き゚	びだくおん き ぎ	鼻濁音
-記号	く゚	びだくおん く ぐ	鼻濁音
-記号	け゚	びだくおん け げ	鼻濁音
-記号	こ゚	びだくおん こ ご	鼻濁音
-記号	カ゚	びだくおん か が	鼻濁音
-記号	キ゚	びだくおん き ぎ	鼻濁音
-記号	ク゚	びだくおん く ぐ	鼻濁音
-記号	ケ゚	びだくおん け げ	鼻濁音
-記号	コ゚	びだくおん こ ご	鼻濁音
+記号	あ゙	あ あ" あ゛ だくてん	濁点付き仮名
+記号	ぁ゙	あ あ" あ゛ ぁ ぁ" ぁ゛ だくてん	濁点付き仮名
+記号	い゙	い い" い゛ だくてん	濁点付き仮名
+記号	ぃ゙	い い" い゛ ぃ ぃ" ぃ゛ だくてん	濁点付き仮名
+記号	ぅ゙	う う" ゔ ぅ ぅ" ぅ゛ だくてん	濁点付き仮名
+記号	え゙	え え" え゛ だくてん	濁点付き仮名
+記号	ぇ゙	え え" え゛ ぇ ぇ" ぇ゛ だくてん	濁点付き仮名
+記号	お゙	お お" お゛ だくてん	濁点付き仮名
+記号	ぉ゙	お お" お゛ ぉ ぉ" ぉ゛ だくてん	濁点付き仮名
+記号	っ゙	づ っ っ" っ゛ だくてん	濁点付き仮名
+記号	な゙	な な" な゛ だくてん	濁点付き仮名
+記号	に゙	に に" に゛ だくてん	濁点付き仮名
+記号	ぬ゙	ぬ ぬ" ぬ゛ だくてん	濁点付き仮名
+記号	ね゙	ね ね" ね゛ だくてん	濁点付き仮名
+記号	の゙	の の" の゛ だくてん	濁点付き仮名
+記号	ま゙	ま ま" ま゛ だくてん	濁点付き仮名
+記号	み゙	み み" み゛ だくてん	濁点付き仮名
+記号	む゙	む む" む゛ だくてん	濁点付き仮名
+記号	め゙	め め" め゛ だくてん	濁点付き仮名
+記号	も゙	も も" も゛ だくてん	濁点付き仮名
+記号	や゙	や や" や゛ だくてん	濁点付き仮名
+記号	ゃ゙	や や" や゛ ゃ ゃ" ゃ゛ だくてん	濁点付き仮名
+記号	ゆ゙	ゆ ゆ" ゆ゛ だくてん	濁点付き仮名
+記号	ゅ゙	ゆ ゆ" ゆ゛ ゅ ゅ" ゅ゛ だくてん	濁点付き仮名
+記号	よ゙	よ よ" よ゛ だくてん	濁点付き仮名
+記号	ょ゙	よ よ" よ゛ ょ ょ" ょ゛ だくてん	濁点付き仮名
+記号	ら゙	ら ら" ら゛ だくてん	濁点付き仮名
+記号	り゙	り り" り゛ だくてん	濁点付き仮名
+記号	る゙	る る" る゛ だくてん	濁点付き仮名
+記号	れ゙	れ れ" れ゛ だくてん	濁点付き仮名
+記号	ろ゙	ろ ろ" ろ゛ だくてん	濁点付き仮名
+記号	わ゙	わ わ" わ゛ だくてん	濁点付き仮名
+記号	ゐ゙	うぃ う"ぃ うぃ" うぃ゛ ゔぃ い い" い゛ だくてん	濁点付き仮名
```

Comment:
I think little people use this but not many people can guess how to pronounce it because ゔぃ is much popular.
This is why "うぃ゛" exists here.
"ゐ" will be re-added.

Reply:
Indeed, I'd like to minimize the ways to type those characters. I think even a single way is fine for people who want to type those characters. Once they know how to type them, they would use that way. I don't like to expand the combinations for just in case.

Reply:
Reduced.